pkg/core/native/native_neo.go

```diff
@@ -113,6 +109,10 @@ func NewNEO() *NEO {
 	md = newMethodAndPrice(n.getNextBlockValidators, 100000000, smartcontract.AllowStates)
 	n.AddMethod(md, desc, true)
 
+	desc = &manifest.Method{Name: "onPersist", ReturnType: smartcontract.VoidType}
+	md = newMethodAndPrice(getOnPersistWrapper(n.onPersist), 0, smartcontract.AllowModifyStates)
+	n.AddMethod(md, desc, false)
```

Comment:
The same thing here, why don't we use the one presented in Manifest.ABI?

Reply:
Fixed.